src/components/Header/Header.jsx

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
-// import toody from '../../images/toody.svg';
-// ^^ this is the Toody logo; uncomment he above line to use it
-import './Header.css';
+import toody from '../../images/toody.svg';
+import styles from './Header.module.css';
 
 const Header = () => {
```

Comment:
[nit] when our component is just the return directly (no computations or other state involved), we don't need the `return` keyword in our return

```JavaScript
const Header = () => (
    <div className={styles.header}>
        .........
    </div>
)
```